data-products/_docs/product-specification.md

```diff
@@ -0,0 +1,75 @@
+# Data product specification
+
+Use the `00-specification.yml` file in your data product definition folder to provide an overall title and description for your data product, and tags to help users find it. If you wish, you can also write a short value proposition outling _why_ the product exists and what data analysis or visualisation might arise from it.
+
+You should also provide contact information for the product.
+
+## Example
+
+```yaml
+specification:
+  data-platform.justice.gov.uk/title: "Example Data Product"
+  data-platform.justice.gov.uk/description: "Example Data Product contains published prison population from 2001 to present"
+  data-platform.justice.gov.uk/product-tags: ["HMPPS", "prisons", "published data", "national statistics"]
+  data-platform.justice.gov.uk/value-proposition: "Adding published prison population allows for a reusable, consistent, safely sharable resource for this commonly used data"
+  data-platform.justice.gov.uk//limitations:
+  data-platform.justice.gov.uk/product-version: 0.0.1
+  data-platform.justice.gov.uk/owner-github-team: "data-platform-admins"
+  data-platform.justice.gov.uk/type: "dataset"
+  support:
+    data-platform.justice.gov.uk/owner-email: "data-platform@digital.justice.gov.uk"
+    data-platform.justice.gov.uk/owner-ms-teams:
+    data-platform.justice.gov.uk/owner-slack-channel: "data-platform"
+    data-platform.justice.gov.uk/product-guidance-url:
+```
+
+## Notes
+
+### Naming and tagging
+
+The product's `title` must be descriptive and unique - we check for uniqueness when data product requests are created.
```

Comment:
Why does it need to be unique?

Reply:
1. Because its a directory `https://github.com/ministryofjustice/data-platform-products/tree/main/data-products`. E.g. 'The Alex Super Product' becomes `./data_products/the_alex_super_product/`
2. It will likely be the root of the API endpoint `https://data-platform.service.justice.gov.uk/api/the-alex-super-product/`
3. Because users may see it listed in a directory of products...
4. ...or may know a product title well enough to search for it directly

It's all hypothetical of course but makes sense to me. I'll add the above to the doc.